### ISSUE TITLE
The patch don't work on PSP and PSVITA(Adrenaline)

### ISSUE THREAD

**Issue by mcamacho97** [fsantecchia/Suikoden-PSP-translation-project#4]
The game doesn't show any type of label, it's empty all the text boxes and it can be played.

**Comment by fsantecchia** [comment]
yeah im sorry you need more tools/files to get to the final version but the google translation is really bad, but just wait a little bit and the team will release a new version of the eng version! this github repo was created to help the translators, you cannot build the game using only the stuff that we have here.
